service/service.go

```diff
@@ -108,6 +89,60 @@ func (svc *Service) Init(ctx context.Context, cfg *config.Config, buildTime, git
 	return nil
 }
 
+// initClients initializes external service clients based on the service configuration.
+// It sets up clients for Zebedee, DatasetAPI, and Topic services if their corresponding flags are enabled.
+func (svc *Service) initClients(ctx context.Context) {
+	if svc.Cfg.EnableZebedeeCallbacks {
+		svc.ZebedeeCli = GetZebedee(svc.Cfg)
+	} else {
+		log.Info(ctx, "healthchecks skipped for zebedee as callbacks disabled")
```

Comment:
This bit is not about healthchecks but about initialising the client.
```suggestion
		log.Info(ctx, "zebedee client not initialised as callbacks disabled")
```